frontend/src/app-components/tables/OverlayTemplate.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2025 Hexastack. All rights reserved.
+ *
+ * Licensed under the GNU Affero General Public License v3.0 (AGPLv3) with the following additional terms:
+ * 1. The name "Hexabot" is a trademark of Hexastack. You may not use this name in derivative works without express written permission.
+ * 2. All derivative works must include clear attribution to the original creator and software, Hexastack and Hexabot, in a prominent location (e.g., in the software's "About" section, documentation, and README file).
+ */
+
+import { Theme } from "@emotion/react";
+import { Alert, AlertProps, Box, SxProps } from "@mui/material";
+import { FC } from "react";
+
+import { useTranslate } from "@/hooks/useTranslate";
+import { TTranslationKeys } from "@/i18n/i18n.types";
+
+import NoDataIcon from "../svg/NoDataIcon";
+
+export const OverlayTemplate = ({
+  i18nKey,
+  icon,
+  color,
+}: {
+  i18nKey: TTranslationKeys;
+  icon?: FC<{ sx?: SxProps<Theme> }>;
+  color?: AlertProps["color"];
+}) => {
+  const Icon = icon || NoDataIcon;
+  const { t } = useTranslate();
+
+  return (
+    <Alert
+      color={color}
+      className="custom-alert"
+      iconMapping={{
+        success: <Icon sx={{ fontSize: "50px" }} />,
+      }}
```

Comment:
This could be simplified. Since the default severity value is 'success'. Using iconMapping could be useful if we intend to make the severity for the Overlay Template configurable I guess.
```suggestion
      icon={<Icon sx={{ fontSize: "50px" }} />}
```

Reply:
100% agreed :magic_wand: